packages/framework-core/src/sensor/health/health-indicators/default-booster-health-indicators.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  BOOSTER_HEALTH_INDICATORS_IDS,
+  BoosterConfig,
+  Class,
+  HealthIndicatorInterface,
+  HealthIndicatorMetadata,
+} from '@boostercloud/framework-types'
+import { BoosterHealthIndicator } from './booster-health-indicator'
+import { BoosterDatabaseHealthIndicator } from './booster-database-health-indicator'
+import { BoosterDatabaseEventsCatalogHealthIndicator } from './booster-database-events-catalog-health-indicator'
+import { BoosterFunctionHealthIndicator } from './booster-function-health-indicator'
+import { BoosterDatabaseReadmodelsCatalogsHealthIndicator } from './booster-database-readmodels-catalogs-health-indicator'
+
+function buildMetadata(
+  config: BoosterConfig,
+  id: BOOSTER_HEALTH_INDICATORS_IDS,
+  name: string,
+  boosterHealthIndicator: Class<HealthIndicatorInterface>
+): HealthIndicatorMetadata {
+  const health = config.sensorConfiguration.health
+  return {
+    class: boosterHealthIndicator,
+    healthIndicatorConfiguration: {
+      id: id,
+      name: name,
+      enabled: health.booster[id].enabled,
+      details: health.booster[id].details,
+      showChildren: health.booster[id].showChildren,
+    },
+  }
+}
+
+/**
+ * Booster configured HealthIndicators
+ */
+export function defaultBoosterHealthIndicators(config: BoosterConfig): Record<string, HealthIndicatorMetadata> {
+  const root = buildMetadata(config, BOOSTER_HEALTH_INDICATORS_IDS.ROOT, 'Booster', BoosterHealthIndicator)
+  const boosterFunction = buildMetadata(
+    config,
+    BOOSTER_HEALTH_INDICATORS_IDS.FUNCTION,
+    'Booster Function',
+    BoosterFunctionHealthIndicator
+  )
+  const boosterDatabase = buildMetadata(
+    config,
+    BOOSTER_HEALTH_INDICATORS_IDS.DATABASE,
+    'Booster Database',
+    BoosterDatabaseHealthIndicator
+  )
+  const databaseEventsCatalog = buildMetadata(
+    config,
+    BOOSTER_HEALTH_INDICATORS_IDS.DATABASE_EVENTS,
+    'Booster Database Events Catalog',
+    BoosterDatabaseEventsCatalogHealthIndicator
+  )
+  const databaseReadModelsCatalogs = buildMetadata(
+    config,
+    BOOSTER_HEALTH_INDICATORS_IDS.DATABASE_READ_MODELS,
+    'Booster Database ReadModels Catalogs',
+    BoosterDatabaseReadmodelsCatalogsHealthIndicator
+  )
+  return {
+    [root.healthIndicatorConfiguration.id]: root,
+    [boosterFunction.healthIndicatorConfiguration.id]: boosterFunction,
+    [boosterDatabase.healthIndicatorConfiguration.id]: boosterDatabase,
+    [databaseEventsCatalog.healthIndicatorConfiguration.id]: databaseEventsCatalog,
+    [databaseReadModelsCatalogs.healthIndicatorConfiguration.id]: databaseReadModelsCatalogs,
```

Comment:
If `boosterDatabase` is a mix of the eventsCatalog and the readModelsCatalogs, Why also including these separately?

Reply:
BoosterDatabase returns details related to the database only (the url right now), and use the events and readmodels catalogs status to calculate the database status, but it will not show information about the catalogs, this is done in each catalog indicator. We need to add them manually to be shown

Reply:
Aha! Thanks